esp/platform/txsummary.hpp

```diff
@@ -51,21 +52,23 @@ class CTxSummary : extends CInterface
     virtual unsigned getElapsedTime() const;
 
     // Appends all summary entries to the given buffer.
-    virtual void serialize(StringBuffer& buffer) const;
+    virtual void serialize(StringBuffer& buffer, const LogLevel logLevel = LogMin) const;
```

Comment:
This, and all of the interface changes in this file, are breaking changes for my security manager plugin. Is this appropriate for a point release, or should it be a minor release?

Reply:
I can change it to a minor release.